src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/ControlDesigner.ControlDesignerAccessibleObjectTests.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using Moq;
+using System.ComponentModel.Design;
+using System.Drawing;
+
+namespace System.Windows.Forms.Design.Tests;
+
+public class ControlDesignerControlDesignerAccessibleObjectTests : IDisposable
```

Comment:
```suggestion
public class ControlDesignerAccessibleObjectTests : IDisposable
```
To improve readability.

Reply:
Please also rename the tests cases.